docs/getting-started.md

```diff
@@ -26,30 +26,135 @@ additional information on the `move` process.
+```
 
 ##### DEPLOY_RAMDISK_URL
 
 This is the URL of the ramdisk to deploy. For example:
 
-`DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"`
+```sh
+  export DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"`
+```
 
 ##### IRONIC_URL
 
 This is the URL of the ironic endpoint. For example:
 
-`IRONIC_URL="http://172.22.0.1:6385/v1/"`
+```sh
+  export IRONIC_URL="http://172.22.0.1:6385/v1/"`
+```
 
 ##### IRONIC_INSPECTOR_URL
 
 This is the URL of the ironic inspector endpoint.
 For example:
 
-`IRONIC_INSPECTOR_URL="http://172.22.0.1:5050/v1/"`
+```sh
+  export IRONIC_INSPECTOR_URL="http://172.22.0.1:5050/v1/"`
+```
+
+##### IRONIC_CA_CERT_B64
+
+This contains the CA certificate for Ironic, encoded in Base 64. It defaults to
+an empty string in case you do not want to provide a CA certificate. Setting
+this variable is optional, but either this variable or `IRONIC_NO_CA_CERT` must
+be set.
+
+```sh
+  export IRONIC_CA_CERT_B64="LS0tLS1C..."
+```
+
+The content of this variable must be string without line wraps. It can be
+generated as follows :
+
+```sh
+  export IRONIC_CA_CERT_B64="$(cat ca.crt | base64 -w 0)"
+```
+
+##### IRONIC_NO_CA_CERT
+
+Do not use a dedicated CA certificate for Ironic API. Any value provided in
+this variable disables additional CA certificate validation. To provide a CA
+certificate, leave this variable unset. If unset, then `IRONIC_CA_CERT_B64`
+must be set.
+
+```sh
+  export IRONIC_NO_BASIC_AUTH="true"
+```
+
+##### IRONIC_USERNAME
+
+Username for Ironic basic auth. Optional, but either this variable or
+`IRONIC_NO_BASIC_AUTH` must be set.
+
+```sh
+  export IRONIC_USERNAME="<username>"
+```
+
+##### IRONIC_PASSWORD
+
+Password for Ironic basic auth. Optional, but either this variable or
+`IRONIC_NO_BASIC_AUTH` must be set.
+
+```sh
+  export IRONIC_PASSWORD="<password>"
+```
+
+##### IRONIC_NO_BASIC_AUTH
+
+Disables basic authentication for Ironic API. Any value provided in this
+variable disables authentication. To enable authentication, leave this variable
+unset. If unset, then `IRONIC_USERNAME` and `IRONIC_PASSWORD` must be set.
+
+```sh
+  export IRONIC_NO_BASIC_AUTH="true"
+```
+
+##### IRONIC_INSPECTOR_USERNAME
+
+Username for Ironic inspector basic auth. Optional, but either this variable or
+`IRONIC_INSPECTOR_NO_BASIC_AUTH` must be set.
+
+```sh
+  export IRONIC_INSPECTOR_USERNAME="<username>"
+```
+
+##### IRONIC_INSPECTOR_PASSWORD
+
+Password for Ironic inspector basic auth. Optional, but either this variable or
+`IRONIC_INSPECTOR_NO_BASIC_AUTH` must be set.
+
+```sh
+  export IRONIC_INSPECTOR_PASSWORD="<password>"
+```
+
+##### IRONIC_INSPECTOR_NO_BASIC_AUTH
+
+Disables basic authentication for Ironic inspector API. Any value provided in
+this variable disables authentication. To enable authentication, leave this
+variable unset. If unset, then `IRONIC_USERNAME` and `IRONIC_PASSWORD` must be
```

Comment:
Should they instead be `IRONIC_INSPECTOR_PASSWORD` and `IRONIC_INSPECTOR_USERNAME` ?

Reply:
good catch